lib/acl-checker.js

```diff
@@ -0,0 +1,230 @@
+'use strict'
+
+const async = require('async')
+const path = require('path')
+const url = require('url')
+
+const DEFAULT_ACL_SUFFIX = '.acl'
+
+class ACLChecker {
+  constructor (options = {}) {
+    this.debug = options.debug || console.log
+    this.fetch = options.fetch
+    this.strictOrigin = options.strictOrigin
+    this.suffix = options.suffix || DEFAULT_ACL_SUFFIX
+  }
```

Comment:
If you're curious about the progress (or rdfjs compatibility for rdflib), here's the tracking issue: https://github.com/linkeddata/rdflib.js/issues/137